### PR TITLE
server, store: speed up server test

### PR DESCRIPTION
**Pull request by zimulala**
### What problem does this PR solve?

Issue Number: close ##24832 

Problem Summary:

### What is changed and how it works?

What's Changed:

Make `CopNextMaxBackoff` is reduced from the 20s to 2ms.

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

| branch | `TestTiFlashFallback` | go test server package with race( exec `go test -v -vet=off -p 5 -timeout 20m -race` / `CI (Race Test Chunk #4)` locally)  |  
----------------------------------------------------------- |----------- |----------- |
 |   before this PR  | 211.016s |     360.423s     |
 |   after this PR  | 5.906s |     192.701s      |

### Release note 

-  `No release note`.

### CODE REVIEW

File: store/copr/coprocessor.go [hunk 1]
```diff
@@ -612,7 +612,7 @@ func chooseBackoffer(ctx context.Context, backoffermap map[uint64]*Backoffer, ta
 	if ok {
 		return bo
 	}
-	newbo := backoff.NewBackofferWithVars(ctx, copNextMaxBackoff, worker.vars)
+	newbo := backoff.NewBackofferWithVars(ctx, CopNextMaxBackoff, worker.vars)
```
Comment (XuHuaiyu) [inline store/copr/coprocessor.go:615]: why not use failpoint ?

Comment (zimulala) [reply store/copr/coprocessor.go:615]: I don't think it's particularly necessary to create a new "failpoint". This is just changing the visibility
Why do you think "failpoint" is better?

Comment (XuHuaiyu) [reply store/copr/coprocessor.go:615]: To avoid the variant be modified unexpectedly in the future.

**Comment by XuHuaiyu** [comment]
```

[2021-05-24T17:04:24.838Z] # github.com/pingcap/tidb/server

[2021-05-24T17:04:24.838Z] vet: server/conn_test.go:779:25: CopNextMaxBackoff not declared by package copr

[2021-05-24T17:04:24.838Z] make: *** [vet] Error 1
```

**Pull request merged**